app/src/main/AndroidManifest.xml

```diff
@@ -387,6 +387,7 @@
         </activity>
         <activity
             android:name=".activities.LoginActivity"
+            android:configChanges="orientation|screenLayout|screenSize|layoutDirection"
```

Comment:
Can you please verify if screenLayout, screenSize and layoutDirection are necessary for this.

Reply:
I believe most of these should be useful. For example, if a user switches the display resolution (eg from 1080p to 1440p) while filling out the login form, the login page is reloaded similarly to how changing the display orientation reloads the page and loses the login details.

Reply:
That makes sense. Could you update the title and description for the PR to reflect the same details. You can also add them in #1368